manim/animation/creation.py

```diff
@@ -76,15 +166,25 @@ def construct(self):
         "lag_ratio": 1,
     }
 
-    def get_bounds(self, alpha):
+    def _get_bounds(self, alpha):
         return (0, alpha)
 
 
 class Uncreate(ShowCreation):
+    """Like :class:`ShowCreation` but in reverse.
+
+    See Also
+    --------
+    :class:`ShowCreation`
+
+    """
+
     CONFIG = {"rate_func": lambda t: smooth(1 - t), "remover": True}
 
 
 class DrawBorderThenFill(Animation):
+    """Show the border first and then show the fill."""
```

Comment:
"Show" seems somewhat inaccurate, maybe trace or actually draw?

```suggestion
    """Draw the border first and then show the fill."""
```